cmd/performance-profile-creator/cmd/root.go

```diff
@@ -17,28 +17,159 @@
 package cmd
 
 import (
+	"context"
 	"fmt"
+	"io/ioutil"
 	"os"
+	"path/filepath"
 
+	"github.com/golang/glog"
 	"github.com/spf13/cobra"
 	yaml "gopkg.in/yaml.v2"
 
 	"k8s.io/utils/pointer"
 
 	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
+	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	clientmachineconfigv1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
+	v1 "k8s.io/api/core/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/component-helpers/scheduling/corev1"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type ClientSet struct {
```

Comment:
why do not use the controller-runtime client?

Reply:
I needed to access the machineconfig client to be able to get mcp so created a user defined clientset something like was done [here](https://github.com/openshift/machine-config-operator/blob/master/test/e2e/framework/clientset.go). Anyway, based on discussion with @fromanirh  we are going to obtain the node/mcp info from the must gather rather than by querying the cluster so we won't need the client. Will add a commit for that.

Reply:
Just for the future, we do not use any clientset under the PAO, we only register the relevant scheme(https://github.com/openshift-kni/performance-addon-operators/blob/6c9616d44117b10f7df1fd799c2dff9a5821f083/main.go#L123), and after use the controller-runtime to get objects(https://github.com/openshift-kni/performance-addon-operators/blob/6c9616d44117b10f7df1fd799c2dff9a5821f083/controllers/resources.go#L29).

Reply:
Great, thanks!!